lib/schema-fs-loader.js

```diff
@@ -43,7 +28,7 @@ const loadSchema = (filepath, context, callback) => {
     const name = path.basename(filepath, '.schema');
     let exports;
     try {
-      exports = processSchemaFile(filepath, decorators, source, context);
+      exports = processSchema(name, source, context);
```

Comment:
Should we maybe keep the `decorators` argument in case anyone wants to specify custom decorators? It can also be optional.

Reply:
At this point I don't see any reason to have it (or need to specify custom decorators), I think we can easily add it back if we ever need it.

Reply:
It will enable the usage of the decorators declared outside of this library by the users of this package.
This can also help with the frequent release of new versions of this package whenever we add or change decorators here.